CONTRIBUTING.md

```diff
@@ -11,17 +11,17 @@ In order to achieve that we have a strong focus on maintainability and high test
 
 - We expect new or modified unit test for every change (written in [Spock](http://spockframework.org/)).
 
-- Your code should be formatted with our IntelliJ [graphql-java-code-style](graphql-java-code-style.xml). 
+- Your code should be formatted with our IntelliJ [graphql-java-code-style](graphql-java-code-style.xml).
 
-- We don't add a new dependency to graphql-java: dependency conflicts will make adaption of graphql-java harder for users, 
```

Comment:
I am confused by this diff: did you change the line break or so?

Reply:
oh yes, that is my vim config killing trailing spaces by default:

![image](https://user-images.githubusercontent.com/5798698/129442866-814e2a81-5141-46bc-96db-8efb8272d596.png)